tools/src/main/java/com/graphhopper/tools/Measurement.java

```diff
@@ -510,4 +528,79 @@ private void store(FileWriter fileWriter, String comment) throws IOException {
         }
         fileWriter.flush();
     }
+
+    /**
+     * Writes a selection of measurement results to a single line in
+     * a file. Each run of the measurement class will append a new line.
+     */
+    private void writeSummary(String summaryLocation, String propLocation) {
+        // choose properties that should be in summary here
+        String[] properties = {
+                "graph.nodes",
+                "graph.edges",
+                "measurement.seed",
+                CH.PREPARE + "time",
+                CH.PREPARE + "shortcuts",
+                "routing.distance_mean",
+                "routing.mean",
+                "routing.visited_nodes_mean",
+                "routingCH.distance_mean",
+                "routingCH.mean",
+                "routingCH.visited_nodes_mean",
+                "measurement.timestamp"
+        };
+        try {
```

Comment:
Can we use `try (FileWriter writer = new FileWriter)` to use automated writer.close call?

Reply:
sure